### DISCUSSION TITLE
we have updated the react-native version from V5 to V6 in our existing app using the migration documentation. App crash issue occur for us only on fresh install and after that App starts working properly.

### DISCUSSION THREAD

**Discussion by praveenp-od** [invertase/react-native-firebase#4713] [Q&A] [answered]
we have updated the react-native version from V5 to V6 in our existing app using the migration documentation. App crash issue occur for us only on fresh install and after that App starts working properly.

E/ReactNativeJS: RNFirebase core module was not found natively on Android, ensure you have correctly added the RNFirebase and Firebase gradle dependencies to your android/app/build.gradle file.

See http://invertase.link/android for the android setup guide.
2020-12-10 17:05:44.032 7230-7632/? E/ReactNativeJS: Module AppRegistry is not a registered callable module (calling runApplication)

--------- beginning of crash
2020-12-10 17:05:44.036 7230-7633/? E/AndroidRuntime: FATAL EXCEPTION: mqt_native_modules Process: com.app, PID: 7230
com.facebook.react.common.JavascriptException: RNFirebase core module was not found natively on Android, ensure you have correctly added the RNFirebase and Firebase gradle dependencies to your android/app/build.gradle file.

See http://invertase.link/android for the android setup guide.,
stack: t@573:619
@573:2091
v@2:1474
@572:127
v@2:1474
@571:243
v@2:1474
@516:271
v@2:1474
@480:246
v@2:1474
@341:226
v@2:1474
@6:203
v@2:1474
d@2:876
global code@1992:4

Please help needed, we tried all the possible way. The error msg "**RNFirebase core module was not found natively on Android, ensure you have correctly added the RNFirebase and Firebase gradle dependencies to your android/app/build.gradle file.**" it's a old lib content still showing. But In our code base we completely removed the V5.

**Comment by praveenp-od** [comment, marked as answer]
<img width="1324" alt="Screenshot 2020-12-25 at 8 36 59 AM" src="https://user-images.githubusercontent.com/37923062/103116519-c803f380-468c-11eb-93ce-ce5029545ffc.png">

There is no code related to V5 integration still we got the Error msg

**Reply by mikehardy** [reply]
This is a problem report, it should not be a discussion. You should open a new issue and follow the template.

If you can reproduce your problem starting from https://github.com/mikehardy/rnfbdemo/blob/master/make-demo.sh we can examine it as a problem with the module, otherwise I have to assume you have a project-specific issue and we can't really debug those for you

Please note we are not on v6 anymore, it is just that v5 to v6 update was the most difficult upgrade, after that they are all very small changes even if they were major versions. We are on v10.3.1 now

**Reply by praveenp-od** [reply]
Sure will test the same in latest version and create an issue if existing,

Thanks
